packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -413,20 +447,44 @@ export default decorate([
           value: 'success',
         },
       ],
-      defaultFilter: ({ countByStatus }) => {
-        if (countByStatus.pending > 0) {
-          return 'pending'
-        }
+      status: ({ _status, countByStatus }) =>
+        defined(_status, () => {
+          if (countByStatus.pending > 0) {
+            return 'pending'
+          }
 
-        if (countByStatus.failure > 0) {
-          return 'failure'
+          if (countByStatus.failure > 0) {
+            return 'failure'
+          }
+
+          if (countByStatus.interrupted > 0) {
+            return 'interrupted'
+          }
+
+          return 'all'
+        }),
+      searchBarFilters: ({ log }) => {
+        const { srs = [], remotes = [] } = log.tasks.find(
+          ({ data = {} }) =>
+            data.type === 'VM' || data.type === 'xo' || data.type === 'pool'
```

Comment:
1. assigning a default value on a destructed object instead of testing if it's undefined  is a practice. Should we ban it? Why?
2. it's worth to create an array and check if the element exists on each loop instead of conditions? Why?

Reply:
```js
({ data : { type } = {}) => ['VM', 'xo', 'pool'].includes(data.type)
```